examples/bazel_env_test.sh

```diff
@@ -88,12 +88,17 @@ Tools available in PATH:
   * pnpm:        @pnpm
   * python:      \$(PYTHON3)
   * python_tool: :python_tool
+  * cargo:       \$(CARGO)
+  * rustfmt:     \$(RUSTFMT)
+  * rustc:       \$(RUSTC)
+  * rustdoc:     \$(RUSTDOC)
```

Comment:
Could you also add `assert_cmd_output` for these tools (and the toolchain) further below? Otherwise our test coverage will be incomplete for rust.

Reply:
oh sorry about that, didn't even look, one moment

Reply:
added in [3d4a6e7](https://github.com/buildbuddy-io/bazel_env.bzl/pull/24/commits/3d4a6e73cf240c0ed03a40d19b8d0e0c076610c7)